src/hello-world/exercise.md

```diff
@@ -0,0 +1,27 @@
+---
+minutes: 5
+existing course material:
+- hello-world.md
+- hello-world/small-example.md
+- exercises/day-1/morning.md
+- exercises/day-1/afternoon.md
+- exercises/day-2/morning.md
+- exercises/day-2/afternoon.md
+- exercises/day-3/morning.md
+- exercises/day-3/afternoon.md
+---
+
+# Exercise: Hello, World
+
+Open the [Rust Playground](https://play.rust-lang.org/). This exercise is mostly
+getting you accustomed to the playground and experimenting with Rust code. 
+
+If you haven't used the site before, you will see a simple hello-world program.
+Try running it.
+
+- Add some variables and print them using the `{..}` notation seen in the
+  lessons.
+- Under "Tools", use the `rustfmt` option to format your code in the "standard"
+  way.
+- If you're interested, use "ASM" under "..." to see the generated assembly
+  code.
```

Comment:
I'm not sure this exercise has enough of a goal condition to be productive for most students. It's good to introduce the playground, but without a purpose the students don't get a sense of achievement and may not have anything they particularly want to try. If we want to cover formatting syntax in detail or demonstrate playground features, those seem like things that are better to do as directed teaching than hoping students stumble upon them in an open-ended exercise.

Reply:
I replaced this with a slide introducing the playground. It gives an instructor a chance to encourage students to keep it open and noodle around during the course, which I think helps students learn and ask good questions.